jenkins/opensearch-ruby/Jenkinsfile

```diff
@@ -39,7 +39,8 @@ pipeline {
                         '''
                     }
                     withAWS(role: 'opensearch-bundle', roleAccount: "${AWS_ACCOUNT_PUBLIC}", duration: 900, roleSessionName: 'jenkins-session') {
-                        s3Upload(file: 'dist', bucket: "${ARTIFACT_BUCKET_NAME}", path: "")
+//                      s3Upload(file: 'dist', bucket: "${ARTIFACT_BUCKET_NAME}", path: "${JOB_NAME}/${OPENSEARCH_RUBY_RELEASE_BRANCH_NAME}/builds")
+                        sh('echo Uploading to s3://${ARTIFACT_BUCKET_NAME}/${JOB_NAME}/${OPENSEARCH_RUBY_RELEASE_BRANCH_NAME}/builds')
```

Comment:
Not sure if we should add that since one release branch should only have one artifact created for it.

Reply:
This changed is required for my approval.  Everything in S3 should be considered read-only, otherwise we can lose changes, by including the build number protect this job from itself.